js_modules/dagit/packages/core/src/assets/AssetEventDetail.stories.tsx

```diff
@@ -0,0 +1,61 @@
+import {MockedProvider} from '@apollo/client/testing';
+import {Box} from '@dagster-io/ui';
+import React from 'react';
+
+import {createAppCache} from '../app/AppCache';
+import {StorybookProvider} from '../testing/StorybookProvider';
+import {WorkspaceProvider} from '../workspace/WorkspaceContext';
+
+import {AssetEventDetail, AssetEventDetailEmpty} from './AssetEventDetail';
+import {
+  BasicObservationEvent,
+  MaterializationEventFull,
+  MaterializationEventMinimal,
+  MaterializationUpstreamDataEmptyMock,
+  MaterializationUpstreamDataFullMock,
+} from './AssetEventDetail.mocks';
+
+// eslint-disable-next-line import/no-default-export
+export default {component: AssetEventDetail};
+
+export const EmptyState = () => {
+  return (
+    <Box style={{width: '950px', display: 'flex', flexDirection: 'column'}}>
```

Comment:
Are the flex properties necessary on the wrapper in all cases? Should that style be pushed down into `AssetEventDetail` itself?

Reply:
Ahh let me see if I can remove the display: flex on here, good call. I'd just added the width to better simulate these appearing in a list + detail view.